### PR TITLE
fix(binders): remove code duplication

### PR DESCRIPTION
**Pull request by stalniy**
Reused existing binders and removed code duplication

### CODE REVIEW

**Comment by stalniy** [comment]
@Leeds-eBooks any feedback on this one?

File: src/binders.coffee [hunk 1]
```diff
@@ -2,10 +2,8 @@
 
 # Sets the element's text value.
 Rivets.public.binders.text = (el, value) ->
-  if el.textContent?
-    el.textContent = if value? then value else ''
-  else
-    el.innerText = if value? then value else ''
+  textProperty = if el.textProperty? then 'textContent' else 'innerText'
```
Comment (jccazeaux) [inline src/binders.coffee]: I think you meant
```javascript
textProperty = if el.textContent?
```

Comment (stalniy) [inline src/binders.coffee]: right. Thanks!

File: src/bindings.coffee [hunk 2]
```diff
@@ -156,6 +156,9 @@ class Rivets.Binding
     else
       Rivets.Util.getInputValue el
 
+  attributeName: =>
+    @attrName or= [@view.prefix, @type].join('-').replace '--', '-'
```
Comment (Leeds-eBooks) [inline src/bindings.coffee]: As I've mentioned before I'm no coffeescript expert but it looks like you're caching the attribute name under `attrName` as a property on the `Binding` instance, is that right? Feels like pollution to add both `attributeName` and `attrName` to all `Binding` instances just to save some code duplication, could you use a closure for caching the value instead?

**Pull request closed without merging**